server/routes/images.go

```diff
@@ -0,0 +1,159 @@
+package routes
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"localdev/main/config"
+	"localdev/main/models"
+	"localdev/main/services"
+	"mime/multipart"
+	"net/http"
+	"text/template"
+	"time"
+
+	"google.golang.org/appengine"
+	"google.golang.org/appengine/log"
+
+	"cloud.google.com/go/datastore"
+	"cloud.google.com/go/storage"
+	vision "cloud.google.com/go/vision/apiv1"
+)
+
+func serveError(ctx context.Context, w http.ResponseWriter, err error) {
+	w.WriteHeader(http.StatusInternalServerError)
+	w.Header().Set("Content-Type", "text/plain")
+	fmt.Fprint(w, "Internal Server Error")
```

Comment:
You can also print the actual error, I don't think it leaks any important information but makes debugging easier.

Reply:
I actually made it like this at first (you can see it in the first commit), but then removed it just in case the error could contain some sensitive data